binnacle/android/build.gradle

```diff
@@ -1,11 +1,13 @@
 buildscript {
+    ext.kotlin_version = '1.3.21'
```

Comment:
What's with Kotlin being included?

Reply:
Sorry, that was a carryover from the bluetooth branch. Removed.